README.md

```diff
@@ -1042,6 +1042,33 @@ client.ws.trades(['ETHBTC', 'BNBBTC'], trade => {
 <details>
 <summary>Output</summary>
 
+```js
+{
+  eventType: 'trade',
+  eventTime: 1508614495052,
+  symbol: 'ETHBTC',
+  price: '0.04923600',
+  quantity: '3.43500000',
+  maker: false,
+  tradeId: 2148226
+}
+```
+
+</details>
+
+#### aggregate trades
```

Comment:
Can you use the name of the method, and add it to the list on top of the readme?

Reply:
Done

Reply:
And same thing on this line? Sorry to be annoying :P